README.md

```diff
@@ -452,7 +452,63 @@ you will get this response
 
 ```
 
-## adding product to cart
+
+## Remove a product from your wishlist 
+`post /wishlist/productID`
```

Comment:
i see all end path are mistaken to `POST` you can change them
all that are in this file

Reply:
ok 

thank you

Reply:
I'm done  dalton